app/src/css/_buttons.scss

```diff
@@ -1,3 +1,7 @@
+$delete-btn-icon: '\f187';
```

Comment:
I didn't actually change much here but my edtior did some auto-formatting on save. I just moved some definition for delete button styles into variables for easier re-use in the bulk granule delete modal.

Reply:
Thank you for refactoring the delete button styles.